### PR TITLE
Fix/remove the validation

### PR DESCRIPTION
**Pull request by dianabit**
This offers the possibility to use either the existing validation or add a custom one. It is up to the consumer to pass the boolean `useNativeValidation` if they decide to use the existing validation and the field has `required` set to `true`. 

The existing validation works very similarly to Casebook. I made it that way so we can use it without having to worry about validation in Casebook, but also, if we decide to do our own validation in Casebook, we can.

### CODE REVIEW

File: source/index.jsx [hunk 1]
```diff
@@ -260,23 +220,18 @@ class LabelledRichTextEditor extends React.Component {
...
   render() {
     const { isFocused, isInvalid, isFullScreen,
       value1, value2, modalIsOpen, activeEditor, lockedForm } = this.state;
     const { editor1, editor2 } = this;
-    const { text } = activeEditor === 1 ? value1.document : value2.document;
-    const { id, events, required } = this.props;
+    // const { text } = activeEditor === 1 ? value1.document : value2.document;
+    const { id, events, required, label, useNativeValidation, customErrorMsg } = this.props;
     const activeEl = document.activeElement;
+    const showValidation = isInvalid && required && useNativeValidation;
```
Comment (chic-geek) [inline source/index.jsx:230]: If we're removing the validation from this component and inverting the control of validation to the consumer, do we need all this? Just looking at @mrdaniellewis comment in issue #9 shouldn't we just provide a `prop` to the component of say `isValid` and then that will choose to add the `aria-invalid=true` or `false` to the `contentEditable`? I think I'm missing something so please let me know.

File: source/index.jsx [hunk 2]
```diff
@@ -328,15 +288,11 @@ class LabelledRichTextEditor extends React.Component {
...
-          label="textarea"
-          id={`hidden_textarea_for_${id}`}
-          defaultValue={text}
-          tabIndex="-1"
-        />
+        {showValidation && (
+        <span id={`${id}_error`} className="form-control-invalid">
+          {customErrorMsg}
+        </span>
+        )}
```
Comment (chic-geek) [inline source/index.jsx:295]: Relates to the previous conversation about simplifying this component and inversion of validation control.

**Comment by mrdaniellewis** [comment]
I'd prefer not to put the validation contain in the component.  This is so the consumer has more control over how it displays the message and what it considers to be invalid.

**Comment by dianabit** [comment]
Ok, since everyone prefers no validation in the component, we'll go with that 👍

**Pull request closed without merging**